### ISSUE TITLE
sigma8 and Pk_interpolator

### ISSUE THREAD

**Issue by msyriac** [CobayaSampler/cobaya#132]
Should "sigma8" as interpreted by Cobaya always mean sigma8(z=0)? I'm trying to understand if the following is expected behavior. Say I'm interested in quantities at z=2 in addition to sigma8(z=0). I add requirements:

```
requires: 'CAMBdata', Pk_interpolator: {z: [2], k_max: 5.2 }}
``` 

On the master branch, this will cause "sigma8" to be the value of sigma8 at z=2, not at z=0. Is this intended behavior? (My mild preference would have been that this run produces an error saying that Pk_interpolator at z=0 is not available in order to evaluate sigma8, which is always interpreted as being at z=0.)

Note that instead if I do:
```
requires: 'CAMBdata', Pk_interpolator: {z: [0,2], k_max: 5.2 }}
``` 

then "sigma8" will be the value at z=0 regardless of the order in which the redshifts are specified.

**Comment by cmbant** [comment]
It should probably be changed, cf https://github.com/CobayaSampler/cobaya/pull/128
It may work as expected if you explicitly include sigma8 in your output derived parameters. Probably sigma8 should be handled like fsigma8 (though you can already do this using sigma_R).

**Comment by msyriac** [comment]
Whoops, yes this is a duplicate of #128.  Is it actually possible to get sigma8 from get_sigma_R in a consistent way? sigma8 is defined for 8**h^-1** Mpc, and h would change in the chain, right?

**Comment by cmbant** [comment]
Yes, I think you'd have to get a range of R and interpolate as needed. The easiest thing is probably to specify sigma8 as a derived parameter (as I think in the generator's default output).

**Comment by cmbant** [comment]
Let's try to consolidate in https://github.com/CobayaSampler/cobaya/pull/133. Let me know if it works for you.

**Comment by msyriac** [comment]
Thanks, I'll try that branch out.
